components/tools/OmeroPy/src/omero/util/populate_metadata.py

```diff
@@ -59,6 +60,12 @@
 
 log = logging.getLogger("omero.util.populate_metadata")
 
+warnings.warn(
+    "This module is deprecated as of OMERO 5.4.8. Use the module"
+    " in the omero-metadata project available from"
+    " https://pypi.org/project/omero-metadata/ instead.",
+    DeprecationWarning)
```

Comment:
While testing an import, I just realized that this warning is displayed on every import of `omero.util.populate_metadata`.
Concretely that implies it will show up every invocation of the CLI since the first step is to load the plugins even if they are not registered.

Although harmless, my feeling is that this is fairly annoying in terms of user experience. I will look at moving the deprecation warning under the main classes like `ParsingContext` instead so that the warning happens at run-time and does not create unnecessary noise.

Reply:
Ponderating the above, from https://docs.python.org/2/library/warnings.html#warning-categories, `DeprecationWarning is  ignored by default` on Python 2.7. If we still wanted to reduce the CLI  noise on Python 2.6, https://docs.python.org/2/library/warnings.html#available-context-managers would certainly be a way to silence these warnings.

Reply:
I doubt that Python 2.6 is worth any avoidable effort at all.

Reply:
Perhaps add this:
```
$ grep warnings bin/omero
import warnings
warnings.filterwarnings("ignore", category=DeprecationWarning)
```